doc/dev-test.md

```diff
@@ -1,29 +1,26 @@
 # How to set up a dev/test environment
 
-NOTE: Ursula comes setup for Vagrant to complete dev/test work within.  Please see the [Ursula Vagrant](https://github.com/blueboxgroup/ursula#vagrant) instructions for more details.
+NOTE: Ursula comes setup for Vagrant to complete dev/test work within.  Please see the [Ursula Vagrant](https://github.com/blueboxgroup/ursula/blob/master/doc/dev-test.md#vagrant) instructions for more details.
 
 Ursula also comes with scripts to automatically spin up a test environment inside of openstack vms (OpenStack on OpenStack).
 
 ## First-time workstation setup
 
-Confirm that you have followed the [installation instructions for ursula](https://github.com/blueboxgroup/ursula#installation), it should have installed the required packages:
+Confirm that you have followed the [installation instructions for ursula](https://github.com/blueboxgroup/ursula#installation).
 
-  - Ansible
-  - Nova client
-  - Keystone client
-  - Neutron client
-  - Glance client
-  - Cinder client
+## Vagrant Support
 
-  - openstack credentials in `$HOME/.stackrc` (t):
+Vagrant support is built in. To get started, run:
 
-```bash
-    $ cat $HOME/.stackrc
-    export OS_USERNAME="example-username"
-    export OS_PASSWORD="example-password"
-    export OS_TENANT_NAME="example-project-name"
-    export OS_AUTH_URL="https://openstack-example-domain.com:35357/v2.0/"
-    export SERVICE_TYPE="compute"
+```
+(env2) $ ./bin/run_vagrant
+Usage: run_vagrant TYPE [minimal]
+Where type is one of the following:
+- allinone ( single monolithic openstack server )
+- standard ( two controllers and one compute )
+- swift ( object storage )
+
+(env2) $ ./bin/run_vagrant standard
```

Comment:
Sure, though I was trying to show that it was running under virtualenv. Thoughts?